src/domain/storage/storage-bucket/storage.bucket.resolver.mutations.ts

```diff
@@ -169,4 +169,14 @@ export class StorageBucketResolverMutations {
     };
     return await this.referenceService.updateReference(updateData);
   }
+
+  @UseGuards(GraphqlGuard)
```

Comment:
this mutation does not belong on this module. It is a pure admin mutation, so please put it in the platform/admin module. This will also resolve the circular dependency. 

Also the name should be "adminStorageMigrateIpfsUrls" or something along those lines.